postgresql/resource_postgresql_grant_test.go

```diff
@@ -503,6 +511,25 @@ func TestAccPostgresqlGrantColumns(t *testing.T) {
 					},
 				),
 			},
+			{
+				Config: fmt.Sprintf(testGrant, "test_view", `["test_column_one", "test_column_two"]`, `["SELECT"]`),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("postgresql_grant.test", "objects.#", "1"),
+					resource.TestCheckResourceAttr("postgresql_grant.test", "objects.0", "test_view"),
+					resource.TestCheckResourceAttr("postgresql_grant.test", "columns.#", "2"),
+					resource.TestCheckResourceAttr("postgresql_grant.test", "privileges.#", "1"),
+					resource.TestCheckResourceAttr("postgresql_grant.test", "privileges.0", "SELECT"),
+					func(*terraform.State) error {
+						return testCheckColumnPrivileges(t, dbName, roleName, []string{testViews[0]}, []string{"SELECT"}, []string{"test_column_one"})
+					},
+					func(*terraform.State) error {
+						return testCheckColumnPrivileges(t, dbName, roleName, []string{testViews[0]}, []string{"SELECT"}, []string{"test_column_one", "test_column_two"})
+					},
+					func(*terraform.State) error {
+						return testCheckColumnPrivileges(t, dbName, roleName, []string{testViews[0]}, []string{""}, []string{"test_column_one", "var"})
+					},
```

Comment:
Kinda curious what's going on in this case. From what I can tell, `testCheckColumnPrivileges` will take a set of privileges, columns, & tables, and ensure the proper privilege grants for those tables & columns are returned from state for the given role (hope I'm understanding that correctly)? But I can't quite grok what the empty privilege set + `test_column_one` & `var` test case is for. 

Is it to ensure there _aren't_ grants for that case since the view is defined as `val` instead of `var` [here](https://github.com/Leafly-com/terraform-provider-postgresql/pull/2/files#diff-2104127886f920aa0ab39668914427270de794c2e3174f02e73ad743ad9c118bR184)? Sorry if this is a dumb question, still getting up to speed on the test internals of this provider 😅

Reply:
You are correct that the intent is to ensure there aren't grants. The `var`, though, is apparently just a typo that I copied and pasted (from [here](https://github.com/Leafly-com/terraform-provider-postgresql/blob/4cd064561c27a981a9af54f71100027cb589bf2b/postgresql/resource_postgresql_grant_test.go#L467)) without noticing it. Which means if this were a test that the privileges _were_ granted, it would fail incorrectly! 😄 

I'll update those typos.

Reply:
Ahhhh ok, so the column name doesn't actually matter. Just that it's asking about a column that isn't granted further up. Ok I think I get it now, cool thank you!

Reply:
I mean, it kinda sorta matters, conceptually, in that the error being thrown would be a "column doesn't exist" error rather than a "permission denied" error, it's just that the test helper doesn't distinguish between errors. So it's definitely more proper to have the correct name.

Reply:
True, that's a fair point